utilities/cloudharness_utilities/deployment-configuration/value-template.yaml

```diff
@@ -71,3 +71,16 @@ harness:
       limits:
         memory: "2Gi"
         cpu: "1000m"
+    backup:
```

Comment:
Better move to global context (see above)

Reply:
@filippomc  Do you want the resources definition as well to be globally configured? It might be useful to have it on the application level, since resources might be modified depending on specific DB backup tool and DB size.

Reply:
I don't see much on application level. Maybe something depending on the type of database we are doing the backup, but a common sizing that fits on every db would be probably enough

Reply:
Okay, I will move it to the global level